lib/fizzbuzz_advanced.rb

```diff
@@ -0,0 +1,24 @@
+class FizzbuzzAdvanced
+  def initialize
+    @values = {
+      3 => "Fizz",
+      5 => "Buzz",
+      7 => "Sivv",
+      13 => "Grrr"
+    }
+  end
+
+  def calculate(num)
+    to_return = ""
+    output = ""
+
+    @values.each do |k, v|
+      output += v if num % k == 0
+    end
+
+    output.empty? ? to_return += num.to_s : to_return += output
```

Comment:
I didn't want the `calculate` method to actually print anything out...if the user wanted to, they could `puts fizz.calculate(25)` or something. I needed `to_return` to store all of the `output`s of each calculation, so `to_return` is the master list of everything. There are probably other ways to build this out...